data/countries/CL.yaml

```diff
@@ -76,10 +92,24 @@ holidays:
                 name:
                   en: Nativity of Liberator Bernardo O'Higgins
                   es: Nacimiento del Prócer de la Independencia
+                active: 
+                  - to: "2017-08-19"
+      NU:
+        name: Ñuble
+        regions:
+          CH:
+            name: Chillán y Chillán Viejo
+            days:
+              08-20:
+                name:
+                  en: Nativity of Liberator Bernardo O'Higgins
+                  es: Nacimiento del Prócer de la Independencia
+                active: 
+                  - from: "2017-08-19"
       TA:
         name: Tarapacá
         days:
-          08-10:
+          "2015-08-10":
             name:
               en: Saint Lawrence
               es: San Lorenzo de Tarapacá
```

Comment:
According to https://rutas.bienes.cl/wp-content/uploads/2020/03/RP63.pdf page 9 the holiday seams to be celebrated once a year (at least in 2020). Maybe this is not a public holiday in the region. If so it should be attributed as "type: observance"

Reply:
According to https://www.bcn.cl/leychile/navegar?idNorma=1080236, the date was a one-off holiday on 2015, but the festivities are celebrated yearly. Do you want to proceed with "observance" every year? do I duplicate to specify the exception?

Reply:
I'd opt for leaving the holiday as observance like:
```yaml
  08-10:
    name:
      en: Saint Lawrence
      es: San Lorenzo de Tarapacá  
    type: observance
    disable:
      - 2015
  "2015-08-10":
    name:
      en: Saint Lawrence
      es: San Lorenzo de Tarapacá
```

Reply:
Cool, thanks!